assets/js/data/cart/selectors.ts

```diff
@@ -90,11 +90,10 @@ export const getCartMeta = ( state: CartState ): CartMeta => {
 
 /**
  * Retrieves cart errors from state.
- *
- * @param {CartState} state The current state.
- * @return {Array<ResponseError>} Array of errors.
  */
-export const getCartErrors = ( state: CartState ): Array< ResponseError > => {
+export const getCartErrors = (
+	state: CartState
+): Array< ApiErrorResponse > => {
```

Comment:
Based on a recent P2 pdToLP-bQ-p2 the vox pop seems to be in favour of the `ApiErrorResponse[]` syntax over `Array< ApiErrorResponse >`. Up to you if you want to change it here but thought I'd mention it anyway.